### ISSUE TITLE
Doesn't work on iOS

### ISSUE THREAD

**Issue by mulderpf** [Purus/launch_review#6]
My app is on the Apple Store.  Using the default parameters (e.g. none supplied), this definitely doesn't work.  This has simply changed my "Rate on app store" link to an "Exit the app" link.  It doesn't seem to crash either, but I can't be sure.

This was observed on a real Apple iPhone X.

I will update the app and send the actual app ID and try again.

In looking at the iOS code (Android developer here), it very much looks like there is some bad hard-coding going on:

e.g. itms-apps://itunes.apple.com/xy/app/**foo**/id%@?action=write-review
and itms-apps://itunes.apple.com/WebObjects/MZStore.woa/wa/viewContentsUserReviews?type=**Purple+Software**&id=%@&action=write-review

**Comment by KaYBlitZ** [comment]
You can try my fork. I fixed many issues. I opened a PR but it hasn't been looked at yet. In pubspec.yaml use

```
launch_review:
    git: git://github.com/KaYBlitZ/launch_review.git
```

For iOS you have to include the app ID.

**Comment by Purus** [comment]
Thanks for your time on the PR. I will merge it today.

Thanks,

Purusothaman Ramanujam

**Comment by Zazo032** [comment]
If this issue is solved in #5, as it's merged, this issue should be closed.

**Comment by Purus** [comment]
Thanks for the issue and creating the PR. It's now available as part of v2.0.0 package.

Feel free to create a new issue if you see any other issues.
